inception/embedding_service.py

```diff
@@ -1,12 +1,8 @@
 import asyncio
 import time
-
-import torch
-
-torch.set_float32_matmul_precision("high")
```

Comment:
This line triggered many instances of [E402 Module level import not at top of file](https://docs.astral.sh/ruff/rules/module-import-not-at-top-of-file/). I have moved it after the import block, which seems safe to do.